src/integrationTest/java/com/mongodb/hibernate/junit/InjectMongoCollection.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.junit;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Target(FIELD)
+@Retention(RUNTIME)
+public @interface InjectMongoCollection {
+    String name();
```

Comment:
Let's rename it to `value` to be able to use `@InjectMongoCollection("books")`, which seems better than `@InjectMongoCollection(name = "books")`.

Reply:
My first iteration is to use `value` as you thought, but my second thought lead me to think explicit element name spec might be more accurate and less confusing. The `name` conveys that the value provided is collection name, not other things.

A perfect solution would be Spring's `@AliasFor`, but we don't have such luxurary.

So let us use `value` then. I over thought previously.